samples/unit-tests/stock-tools/gui/demo.js

```diff
@@ -61,3 +61,115 @@ QUnit.test('Stocktools GUI', function (assert) {
     );
     */
 });
+
+QUnit.test('Disabling and Enabling stock Tools buttons, when series are invisible, #14192', function (assert) {
+    var wasInitCalled = false;
+
+    // Creating test controller and adding styles to button.
+    var toolsContainer = document.createElement('div');
+    toolsContainer.className += 'tools-container';
+    var button = document.createElement('button');
+    button.className += 'dummy-button';
+    button.innerHTML = 'dummy button';
+
+    toolsContainer.appendChild(button);
+
+    document.getElementById('container').parentNode.insertBefore(toolsContainer, document.getElementById('container'));
+
+    var chart = Highcharts.stockChart('container', {
```

Comment:
Not a mistake, but it's a better practice to keep the `var` declarations on top of the function scope. You could also use `const` and `let`.